packages/dd-trace/src/plugins/util/git.js

```diff
@@ -17,15 +18,22 @@ const {
   CI_WORKSPACE_PATH
 } = require('./tags')
 
+const GIT_REV_LIST_MAX_BUFFER = 8 * 1024 * 1024 // 8MB
+
 function getRepositoryUrl () {
   return sanitizedExec('git config --get remote.origin.url', { stdio: 'pipe' })
 }
 
 function getLatestCommits () {
-  return execSync('git log --format=%H -n 1000 --since="1 month ago"', { stdio: 'pipe' })
-    .toString()
-    .split('\n')
-    .filter(commit => !!commit)
+  try {
+    return execSync('git log --format=%H -n 1000 --since="1 month ago"', { stdio: 'pipe' })
+      .toString()
+      .split('\n')
+      .filter(commit => !!commit)
```

Comment:
Pretty sure this is equivalent (and at least a little bit faster) to
```suggestion
      .filter(commit => commit)
```